src/clib/pio_config.h.in

```diff
@@ -31,6 +31,9 @@
 /** Set the regex to use to save decomps */
 #define PIO_SAVE_DECOMPS_REGEX "@PIO_SAVE_DECOMPS_REGEX@"
 
+/** Set to non-zero to limit the number of non-contiguous regions cached in a single IO process. */
+#define PIO_LIMIT_CACHED_IO_REGIONS @LIMIT_CACHED_IO_REGIONS@
+
 /** Maximum number of non-contiguous regions cached in a single IO process. */
 #define PIO_MAX_CACHED_IO_REGIONS @PIO_MAX_CACHED_IO_REGIONS@
```

Comment:
What would be the value of PIO_MAX_CACHED_IO_REGIONS when PIO_LIMIT_CACHED_IO_REGIONS is off (0)?

Reply:
When PIO_LIMIT_CACHED_IO_REGIONS is OFF, PIO_MAX_CACHED_IO_REGIONS
would be defined without a value in pio_config.h:
`#define PIO_MAX_CACHED_IO_REGIONS`
This seems OK since PIO_MAX_CACHED_IO_REGIONS will not be used at all in this case.
Do you still want to define its value to 0 if PIO_LIMIT_CACHED_IO_REGIONS is OFF?

Reply:
It would be good to have a default value for PIO_MAX_CACHED_IO_REGIONS in this case (ensure its defined to something in all cases)

Reply:
OK, PIO_MAX_CACHED_IO_REGIONS is set to 0 if PIO_LIMIT_CACHED_IO_REGIONS is OFF.